src/server/mautic/mauticAPI.js

```diff
@@ -47,12 +47,12 @@ export const Mautic = {
       tokens: { code, firstName: user.fullName }
     })
   },
-  sendRecoveryEmail(user: UserRecord, mnemonic: string) {
-    if (!(mnemonic && user.fullName && user.mauticId && Config.mauticRecoveryEmailId))
+  sendRecoveryEmail(user: UserRecord, magicLink: string) {
```

Comment:
@StanislavShevchenko this functionality should be placed in other new function to save old functionality